src/conda_devenv/devenv.py

```diff
@@ -428,62 +433,53 @@ def __call_conda_env_update(
     output_filename: Path,
     env_manager: Literal["conda", "mamba"],
 ) -> str | int | None:
-    command = [
-        env_manager,
+    cmdline = [
```

Comment:
Since it lacks the first argument (command) to be a full command line, maybe it could be called `cmdline_args`?